src/PerformanceTests/Common/Statistics/SimpleStatisticsFeature.V6.cs

```diff
@@ -22,6 +22,8 @@ public SimpleStatisticsFeature()
 
         protected override void Setup(FeatureConfigurationContext context)
         {
+            Console.WriteLine("setup SimpleStatisticsFeature");
```

Comment:
I see this setup being called twice in the `GatedSendLocalRunner` test, is that expected @ramonsmits ?

Reply:
@timbussmann The endpoint is created multiple times so that is probably the reason you see it.